geode-web-management/src/main/java/org/apache/geode/management/internal/rest/ManagementLoggingFilter.java

```diff
@@ -70,7 +70,7 @@ private void logRequest(HttpServletRequest request, ContentCachingRequestWrapper
     }
     String payload = getContentAsString(wrappedRequest.getContentAsByteArray(),
         wrappedRequest.getCharacterEncoding());
-    logger.debug(String.format(requestPattern, request.getMethod(), requestUrl,
+    logger.info(String.format(requestPattern, request.getMethod(), requestUrl,
```

Comment:
Why was this changed back to info?

Reply:
sorry, this was not meant to be checked in